monitoring/sentry.md

```diff
@@ -0,0 +1,214 @@
+# Sentry
+
+1. Check out the official guide of creating a new project in Sentry [admin panel](https://docs.sentry.io/product/sentry-basics/integrate-frontend/create-new-project/).
+
+2. Configure [alert rules](https://uptech-team.sentry.io/alerts/rules/) for your project.
+  ![Alerts configuring](../content/Screenshot 2023-05-16 at 13.04.24.png)
```

Comment:
Screenshots doesn't work
<img width="949" alt="image" src="https://github.com/uptechteam/fe-cookbook/assets/12031031/0c8647e1-7c5e-49eb-ae0e-373abd12a481">

Reply:
Good catch! I'll fix it